test/e2e/common/container_probe.go

```diff
@@ -414,9 +417,12 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		framework.ExpectNoError(err)
 
 		framework.Logf("Container started at %v, pod became ready at %v", startedTime, readyTime)
-		if readyTime.Sub(startedTime) < 40*time.Second {
+		if readyTime.Sub(startedTime) < sleepBeforeStarted*time.Second {
```

Comment:
note, the original code gave extra 5 seconds gap to account for potential fast bash execution or slow probe manager initialization